pyop/pyfunc.cc

```diff
@@ -377,7 +377,7 @@ void PyCustomOpDef::AddOp(const PyCustomOpDef* cod) {
   // No need to protect against concurrent access, GIL is doing that.
   auto val = std::make_pair(op_domain, std::vector<PyCustomOpFactory>());
   const auto [it_domain_op, success]  = PyOp_container().insert(val);
-  assert(success || it_domain_op->second.length() > 0);
+  assert(success || it_domain_op->first.length() > 0);
```

Comment:
This statement fails to compile in debug builds because `it_domain_op->second` is a `std::vector`, which does not have a `length()` method. From the surrounding context, I'm assuming that the assert is meant to check the domain string's length.

Reply:
Use `std::string::empty` to check for emptiness not `std::string::legnt() > 0`. The former is more readable than the latter.

Reply:
Good catch.
The assertion checks either the op_domain already exists(the vector is not empty) , or it was newly created. So the 'length' should be replaced with 'size' of std::vector.